### PR TITLE
Add empty state prompt to List page

### LINKED ISSUE #7: 7. As a user, I want to see a welcoming prompt to add my first item if my list is empty to help me get oriented to how the app works.
## Summary

When a user starts a new list, they need a little guidance on how to get started. Think through what you'd want to see as a user and make some intentional user experience decisions that will guide our users in confidently adding their very first item. 

## Acceptance criteria
- [x] The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item

## Notes
- [The wireframe](https://github.com/the-collab-lab/smart-shopping-list/blob/main/_resources/wireframes/list-empty.png) is a little short on the "welcoming" part of the story, but otherwise is functionally complete for the purposes of the AC.

### PR DESCRIPTION
**Pull request by codecaviette**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description
When a user starts a new list, they need a little guidance on how to get started. We added an empty state prompt, clear instructions, and an Add Item button. 

## Related Issue
closes #7. 

## Acceptance criteria
- [x] The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Testing Steps / QA Criteria

1. Navigate to your main branch and pull all changes with `git pull`, then navigate to the feature branch ms-jc-add-getstarted-prompt. 
2. Start up your dev environment by entering `npm start`.
3. Ensure you don't have any list tokens stored in your browser's local storage. You can check by going to Inspect in your browser, tapping Application tab, and clearing any tokens listed.
4. In the application, navigate to the List page where you should see the option to create a new list - tap that button. 
5. You will then see a prompt to add your first item.
6. Click the Add Item button, which will navigate you to the Add Item page. Add your item. 
7. Navigate back to the List page, and you should now see the option to search your list.

### CODE REVIEW

File: src/views/List.jsx [hunk 1]
```diff
@@ -33,30 +40,40 @@ export function List({ data }) {
 		}
 	}, [searchData, data.length]);
 
+	console.log(data);
```
Comment (avinashi10) [inline src/views/List.jsx]: This console.log can be removed before merging

**Review by avinashi10 — APPROVED** [review]
Looks good! The navigation is smooth and clear - great directions too!

**Review by hajschroeder — APPROVED** [review]
Looks good!

**Review by thetrend — APPROVED** [review]

File: src/views/List.jsx [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
 
 export function List({ data }) {
-	// state for input
+	// State for input
 	const [input, setInput] = useState('');
 	const [searchData, setSearchData] = useState(data);
 	const [isValid, setIsValid] = useState(false);
 	const [searchLength, setSearchLength] = useState(1);
-	// Handle event
+	let navigate = useNavigate();
```
Comment (lyleschemmerling) [inline src/views/List.jsx]: can be `const`

**Review by lyleschemmerling — APPROVED** [review]
lgtm

**Review by jeremiahfallin — APPROVED** [review]
Nice work! lgtm.

**Review by thetrend — COMMENTED** [review]
great job!

**Pull request merged**